calc/src/calc_fee.rs

```diff
@@ -63,6 +63,9 @@ impl Multiplier {
 
             ("calamari", _v) => V2(new_u128(inner)),
 
+            ("karura", _v) => V2(new_u128(inner)),
```

Comment:
should be a way to configure this without modify rust code. at least for common cases.

Reply:
Yea we have had discussion about this, especially since parachains are going to want more and more support for fee calculations as time moves on. We thought about a plugin, and some different options. Still something to consider but I agree.